lib_eio_luv/eio_luv.ml

```diff
@@ -1155,6 +1156,14 @@ let rec run : type a. (_ -> a) -> a = fun main ->
             with Luv_error _ as ex ->
               discontinue k ex
           )
+          | Eio_unix.Private.Pipe sw -> Some (fun k ->
+            let r, w = Unix.pipe ~cloexec:true () in 
```

Comment:
libuv has pipes, can we use that ?
https://github.com/aantron/luv/blob/0ed668e87ca55d65f4f84422b0dd3381701c71b0/src/pipe.mli#L40

Reply:
Thanks for the speedy review! :))

Yes, I think so but I had to specify no read and write flags (see [uv_pipe](http://docs.libuv.org/en/v1.x/pipe.html#c.uv_pipe)) so it ended up being

```ocaml
let r, w = Luv.Pipe.pipe ~read_flags:[] ~write_flags:[] () |> or_raise in
```

Do you think that's okay? (They're opened with cloexec `true`).

Reply:
I think it's fine, the only flag it accepts is `nonblock`.